src/template-pages/content-page.html

```diff
@@ -28,7 +28,7 @@
     <meta name="DCTERMS.audience" scheme="AGLSTERMS.agls-audience" content="">
     <meta name="DCTERMS.jurisdiction" scheme="AGLSTERMS.AglsJuri" content="Queensland">
 
-    <!--#include virtual="/assets/includes-cdn/head-assets.html"-->
+    <!--#include virtual="/assets/includes-local/head-assets.html"-->
```

Comment:
This should be cdn

Reply:
I have not changed these . Is something wrong in the build system?

Reply:
oh these are in the source area . I am not sure how these got changed will change it back

Reply:
Its not related to build since it is in src folder. You would have to manually change it back to cdn